cli/utils.ts

```diff
@@ -20,30 +22,29 @@ export type AnalyzedFunction = {
     returns: ValidatorJSON | null
 };
 
-export function getFunctionSpec(prod?: boolean, filePath?: string) {
-    if (filePath && prod) {
+export function getFunctionSpec(prod?: boolean, filePathOpt?: string) {
```

Comment:
nit, filePathOpt and filePath aren't different enough that I know what each of them mean; does opt stand for option? Are there other names that would identify them better?

Reply:
yes, opt means option. i can make this clearer